lib/env.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  ADMIN_REPO: process.env.SAFE_SETTINGS_ADMIN_REPO || 'admin',
+  CONFIG_PATH: '.github',
+  SETTINGS_FILE_PATH: process.env.SAFE_SETTINGS_SETTINGS_FILE_PATH || 'settings.yml',
```

Comment:
@quike Could we simplify the names `SAFE_SETTINGS_ADMIN_REPO` and `SAFE_SETTINGS_SETTINGS_FILE_PATH` to `ADMIN_REPO` and `SETTINGS_FILE_PATH`?

Reply:
@decyjphr sure, I added in that way as they are global env variables to avoid clash with any other existing variable in the system keeping the scope for safe settings only as for example `ADMIN_REPO` is quite generic and could be used by other app or component within a defined system. But let me change it if you prefer it that way.

Reply:
@decyjphr @martinm82 done, can you guys review whenever you can? Thanks

Reply:
Looks good to me.